README.md

```diff
@@ -114,7 +114,7 @@ To format individual files or arbitrary codes from stdin, the `rustfmt` binary s
 examples follow:
 
 - `rustfmt lib.rs main.rs` will format "lib.rs" and "main.rs" in place
-- `rustfmt` will read a code from stdin and write formatting to stdout
+- `rustfmt` will read a code from stdin and write formatting to stdout. When reading from stdin, `rustfmt` will not exit with an error code unless there were parsing errors or interal `rustfmt` errors.
```

Comment:
I wonder if it may be worth just describing the exit code behavior for both scenarios in this section too (the files input mode), as I think it's not just the stdin input mode that could use more clarity.

What do you think?